src/extractors/getFixedTFunction.ts

```diff
@@ -0,0 +1,99 @@
+import * as BabelTypes from '@babel/types';
+import * as BabelCore from '@babel/core';
+import extractTFunction from './tFunction';
+import { ExtractedKey } from '../keys';
+import { Config } from '../config';
+import { getFirstOrNull, evaluateIfConfident } from './commons';
+import { CommentHint, getCommentHintForPath } from '../comments';
+
+/**
+ * Check whether a given CallExpression path is a call to `useTranslation` hook.
+ * @param path: node path to check
+ * @param config: plugin configuration
+ * @returns true if the given call expression is indeed a call to
+ *   `useTranslation`
+ */
+function isGetFixedTFunction(
+  path: BabelCore.NodePath<BabelTypes.CallExpression>,
+  config: Config,
+): boolean {
+  const callee = path.get('callee');
+
+  if (!callee.isMemberExpression()) return false;
+
+  const obj = callee.get('object');
+  if (!obj.isIdentifier()) return false;
+
+  const prop = callee.get('property');
+  if (Array.isArray(prop) || !prop.isIdentifier()) return false;
+
+  return (
+    config.i18nextInstanceNames.includes(obj.node.name) &&
+    prop.node.name === 'getFixedT'
```

Comment:
Wouldn't the [`referencesImport`](https://github.com/gilbsgilbs/babel-plugin-i18next-extract/blob/068bc34c163c0d0292074a8ccff38590451e692a/src/extractors/commons.ts#L85) be preferable here?

Reply:
So I tried that, but it didn't catch something like this:

```ts
import i18n from "i18next";

const t = i18n.getFixedT( ... );
```

I wasn't sure if ppl would be using `i18n` as a variable or pulling the function as `import { getFixedT } from "i18next";` ... but it seems like the docs on i18next website have examples using the former.

The code block itself here is pulled from `i18nextInstance.ts` extractor.

Reply:
Nice catch. What you pointed out is probably a limitation/bug of `referencesImport` that should be fixed in another PR as it may affect other extractors. I'm afraid that this bug comes down to the `referencesImport` implementation from Babel unfortunately :thinking: .

Anyways, after thinking of it, I think it makes sense to use `i18nextInstancesNames` as you did since you get `getFixedT` function from the i18next instance. Let's keep it like this then.

Do you think it would make sense to make `i18nextInstance` and `getFixedTFunction` use a helper function in `commons.ts` instead of copy-pasting this function body?

Reply:
Yea I can do this.